docs/install/installing-istio.md

```diff
@@ -0,0 +1,281 @@
+---
+title: "Installing Istio for Knative"
+weight: 15
+type: "docs"
+---
+
+This guide walks you through manually installing and customizing Istio for use
+with Knative.
+
+If your cloud platform offers a managed Istio installation, we recommend
+installing Istio that way, unless you need the ability to customize your
+installation. If your cloud platform offers a managed Istio installation,
+the [install guide](./) for your specific platform will have those instructions.
+For example, the [GKE Install Guide](./knative-with-gke) includes the
+instructions for installing Istio on your cluster using `gcloud`.
+
+## Before you begin
+
+You need:
+- A Kubernetes cluster created
+- [`helm`](https://helm.sh/) installed
+
+## Installing Istio
+
+When you install Istio, there are a couple of different steps, and a few options
+depending on your goals. For a basic Istio installation suitable for most use
+cases, see the [Default Istio installation](#default-istio-instllation)
+instructions. Those steps will get you up and running quickly without having to
+make decisions about Istio. To customize Istio your Istion installion for use
+with Knative, see the [Custom Istio installation](#customizing-your-installation) instructions.
+
+### Default Istio installation
+The following steps install a default version of Istio that is appropriate for
+most Knative use cases.
+
+1. Enter the following commands to download Istio:
+   ```shell
+   # Download and unpack Istio
+   export ISTIO_VERSION=1.1.3
+   curl -L https://git.io/getLatestIstio | sh -
+   cd istio-${ISTIO_VERSION}
+   ```
+
+1. Enter the following command to install the Istio CRDs first:
+   ```shell
+   for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
+   ```
+   Wait a few seconds for the CRDs to be committed in the Kubernetes API-server,
+   then continue with these instructions.
+
+1. Enter the following command to install Istio:
```

Comment:
@ZhiminXiang 

Is this the basic Istio installation we should recommend?

Bonus points -- is there a way to do our "basic" installation without using `helm`?

Reply:
1. The default command is exactly the same as the command in the section `Installing Istio with sidecar injection`. We should probably keep only one of them.
2. We probably want to recommend users to use the instruction `installing Istio with no sidecar injection` as the default instructions. @tcnghia @mattmoor WDYT?

Reply:
Updated to recommend no sidecar injection.

Reply:
And updated to go back to recommending automatic sidecar injection as the default, since I found info supporting that in the Custom Install guide, which I believe was reviewed by Matt and Nghia.